kubernetes/.openapi-generator-ignore

```diff
@@ -0,0 +1,9 @@
+.gitignore
+.travis.yml
+*gradle*
+*sbt*
+git_push.sh
+README.md
+
+# Remove when changes in kubernetes-client/java#366 make into upstream openapi-generator
+src/main/java/io/kubernetes/client/JSON.java
```

Comment:
@yue9944882 this needs to be changed now that you renamed packages

Reply:
good catch